2-mandatory/1-create-functions.js

```diff
@@ -44,7 +51,14 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arr) {
+  return arr.map((item) => {
+    if (item > 100) {
+      return 100 + "%";
+    } else {
+      return Number(item.toFixed(2)) + "%";
```

Comment:
Because you are doing `+ "%", the `Number` will be turned back into a string - you don't need to cast it.

```suggestion
      return item.toFixed(2) + "%";
```

Reply:
if i dont use Number here my result is 23.00% however i need 23%